### ISSUE TITLE
ahoy.reset() doesn't seem to work

### ISSUE THREAD

**Issue by ndbroadbent** [ankane/ahoy#379]
I rolled back my database and reran the migrations. Then I ran `ahoy.reset()` on the frontend. I reloaded the page and called `binding.pry` in my index action, and checked the value of `current_visit`:

```
> current_visit
  Ahoy::Visit Load (2.1ms)  SELECT  "ahoy_visits".* FROM "ahoy_visits" WHERE "ahoy_visits"."visit_token" = $1 ORDER BY "ahoy_visits"."id" ASC LIMIT $2  [["visit_token", "1e0a9cec-5d3c-4608-9899-f44eac357484"], ["LIMIT", 1]]
  ↳ (pry):2
=> nil
```

Not sure where the `visit_token` is coming from and why it's not being cleared. But it also might be a good idea to check for `nil` and recreate any missing visits.

This might be because I'm using `Ahoy.cookie_domain = :all` and was running `ahoy.reset()` on a subdomain. But I also ran it on the root domain, and a new visit is still not being created.

Note: This is in Firefox.

**Comment by ndbroadbent** [comment]
Damn I think this was a Firefox bug. I just can't get rid of this 'ahoy_visit' cookie, even after trying to manually remove it, and [trying an add-on](https://addons.mozilla.org/en-US/firefox/addon/remove-cookies-button-ver1x/). I've been restarting Firefox and the Rails server, and I can't figure out how to remove the cookie.

**Comment by ndbroadbent** [comment]
Fixed by #380
